torch_ttnn/passes/lowering/to_tt_pass.py

```diff
@@ -385,7 +385,7 @@ def rewrite_node(node):
 
             if node.target == torch.ops.aten.clone.default:
                 arg_metadata = node.meta["val"]
-                ttnn_dtype = torch_dtype_to_ttnn_dtype(arg_metadata.dtype)
+                # ttnn_dtype = torch_dtype_to_ttnn_dtype(arg_metadata.dtype)
```

Comment:
Is this intentional?

Reply:
One of model come across arg_metadata.dtype = torch.int32 and I think no one use `ttnn_dtype` so I comment it

Reply:
Oh this line is to check the support of clone, I'll revert it